libecho/src/BitSender.cc

```diff
@@ -0,0 +1,67 @@
+#include "BitSender.h"
+
+#include <cmath>
+#include <iostream>
+
+const QAudioFormat::Endian OUTPUT_BYTEORDER = QAudioFormat::LittleEndian;
+const int OUTPUT_CHANNEL_COUNT = 1;
+const char *OUTPUT_CODEC = "audio/pcm";
+const int OUTPUT_SAMPLERATE = 44100;
+const int OUTPUT_SAMPLESIZE = 8;
+const QAudioFormat::SampleType OUTPUT_SAMPLETYPE = QAudioFormat::UnSignedInt;
+/**^ QAudioFormat settings we use while sending audio. */
+
+static constexpr size_t SOUNDS_PER_BYTE =
+    8; /**< States the number of sounds we need to play to encode a byte of data. */
```

Comment:
8 in next line looks bad, put comment before this line, and 8 with the rest of declaration